### DISCUSSION TITLE
Rune run

### DISCUSSION THREAD

**Discussion by 29wumya** [Prodigy-Hacking/ProdigyMathGameHacking#1024] [(A) Anything]
Whenever I press the start rune run, this pops up.
![image](https://user-images.githubusercontent.com/80365402/122315989-f04c0c80-cecf-11eb-9970-52d84fc7d093.png)
Does anyone know what is wrong with it

**Comment by 29wumya** [comment]
Pls help if you can

**Comment by TheFinality** [comment]
u either used the runes from get all runes or you used a crystall monster or you used edit pet

**Comment by donaldli2020** [comment]
Once I got it when I was inactive. I used close all popups, and rune runs don't work until I reload.
